config.go

```diff
@@ -338,3 +338,31 @@ func nodejs(c *Config) error {
 
 	return nil
 }
+
+// python config.
+func python(c *Config) {
+	if c.Proxy.Command == "" {
+		c.Proxy.Command = "python app.py"
+	}
+
+	// Only add build & clean hooks if a requiremnts.txt exists
+	if !util.Exists("requirements.txt") {
+		return
+	}
+
+	// Set PYTHONPATH env
+	if c.Environment == nil {
+		c.Environment = config.Environment{}
+	}
+	c.Environment["PYTHONPATH"] = ".pypath/"
+
+	// Copy libraries into .pypath/
+	if c.Hooks.Build == "" {
+		c.Hooks.Build = `mkdir -p .pypath/ && pip install -r requirements.txt -t .pypath/`
```

Comment:
First of all this awesome thanks!

My only question is would doing this each time be idiomatic while developing? For example with Node you'd typically `npm install` outside of Up just to develop anyway, vs doing it on each deploy (ideally). 

I know nothing about the Python ecosystem though, just curious!